apps/src/templates/teacherDashboard/sectionAssignmentTestHelper.js

```diff
@@ -2,16 +2,79 @@ export const fakeTeacherSectionsForDropdown = [
   {
     name: 'Initally Selected Section',
     id: 1,
-    isAssigned: false
+    isAssigned: false,
+    code: 'aaa',
+    courseId: 0,
+    courseOfferingId: 0,
+    courseVersionId: 0,
+    unitId: 0,
+    participantType: 'student'
   },
   {
     name: 'Assigned Section',
     id: 2,
-    isAssigned: true
+    isAssigned: true,
+    code: 'bbb',
+    courseId: 1,
+    courseOfferingId: 1,
+    courseVersionId: 1,
+    unitId: 1,
+    participantType: 'student'
   },
   {
+    // review the courseId and courseofferings onward
```

Comment:
What does this comment mean?

Reply:
Again, a comment to myself that didn't get deleted.  Making a note to self to check more thoroughly to these things in the future